### PR TITLE
Tell users to left align but not justify body copy

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#2075](https://github.com/alphagov/govuk-design-system/issues/2075).

Updates our typography guidance to tell users why they should:

- usually left-align body copy
- avoid 'justifying' blocks of text so that it's aligned to both the left and right margins

### CODE REVIEW

File: src/styles/typography/index.md.njk
Comment (EoinShaughnessy) [inline src/styles/typography/index.md.njk:84]:
```suggestion
Do not 'justify' blocks of body copy so that they're aligned to both the left and right margins. Doing this creates wider spaces between words, which can also make the text hard to read.
```

**Review by davidc-gds — APPROVED** [review]
Looks good to me! I think it would be good to wait for Calvin's review before merging though (not sure if he's seen this work yet)

**Review by calvin-lau-sig7 — APPROVED** [review]

**Pull request merged**
